### PR TITLE
MA-3786: Updated runtime node version to 8.10 and ProjectImage to 8.1…

### PR DESCRIPTION
**Pull request by bhargavapratik**
MA-3786: Updated runtime node version to 8.10 and ProjectImage to 8.11.0.

### CODE REVIEW

File: template.yaml [hunk 1]
```diff
@@ -93,7 +93,7 @@ Resources:
     Properties:
       FunctionName: SungardAS-ConfigRules
       Handler: index.handler
-      Runtime: nodejs4.3
+      Runtime: nodejs8.10
```
Comment (SonalOjha) [inline template.yaml:96]: Make changes to accept this as a param

File: template.yaml [hunk 2]
```diff
@@ -142,7 +142,7 @@ Resources:
     Properties:
       FunctionName: SungardAS-ConfigRules-CloudwatchLogs
       Handler: index_cloudwatchlogs.handler
-      Runtime: nodejs4.3
+      Runtime: nodejs8.10
```
Comment (SonalOjha) [inline template.yaml:145]: Make changes to accept this as a param

**Review by SonalOjha — CHANGES_REQUESTED** [review]

File: template.yaml (hunk 1 above)
Comment (bhargavapratik) [reply template.yaml:96]: Runtime value cannot be a Parameter. still if we need to try this , then we can do while we move code to BitBucket.

**Review by RajkumarShivage — APPROVED** [review]
LGTM

File: template.yaml (hunk 1 above)
Comment (bhargavapratik) [reply template.yaml:96]: MA-4154 Jira for taking Runtime as Parameter.

**Review by SonalOjha — APPROVED** [review]
With changes getting addresses in MA-4154 approving the PR.

**Pull request merged**